packages/block-library/src/image/style.scss

```diff
@@ -157,25 +157,48 @@
 	display: flex;
 	flex-direction: column;
 
+	img {
+		cursor: zoom-in;
+	}
+
 	button {
+		opacity: 0;
 		border: none;
-		background: none;
+		background: #1e1e1e;
 		cursor: zoom-in;
-		width: 100%;
-		height: 100%;
+		width: 25px;
+		height: 25px;
 		position: absolute;
 		z-index: 100;
+		top: 5px;
+		right: 5px;
```

Comment:
Unsure about conventions, but should this be `left` for `rtl` languages?

Reply:
Maybe! That sounds like a good candidate for a follow up PR. I've added this point to the next version in the [lightbox tracking issue](https://github.com/WordPress/gutenberg/issues/51132).